docker/xgb/centos/Dockerfile-centos-gpu

```diff
@@ -25,10 +25,13 @@ ENV JAVA_VERSION '8'
 ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
 ENV PATH ${JAVA_HOME}/bin:${PATH}
 
+# Install Python 3.7
+COPY xgb_build/scripts/Setup.dist.patched /usr/src/
+COPY xgb_build/scripts/install_python_source_centos /usr/sbin/
 RUN \
-    chmod +x /usr/sbin/install_python && \
+    chmod a+x /usr/sbin/install_python_source_centos && \
     sync && \
-    /usr/sbin/install_python
+    /usr/sbin/install_python_source_centos 3.7.0 /usr/src/Setup.dist.patched
 
```

Comment:
H2O-3 support Python 3.6 and up.  Does this means users would need to instsall Python 3.7 and up?

Reply:
No, this is used to build h2oai/xgboost fork for omp, gpu, minimal, and python wheels, we need to meet requirements for the original repo. 

H2O-3 use only xgboost4j (Still java 8) and make its own python and R integration, that will follow our rules for compatibility. 

I know its a bit shady, its still confusing even for my self..

Short answer is no.